indice_pollution/history/models/vigilance_meteo.py

```diff
@@ -174,21 +181,11 @@ def __repr__(self) -> str:
     @classmethod
     def make_max_couleur(cls, vigilances):
         couleurs = [v.couleur_id for v in vigilances]
-        return max(couleurs) if couleurs else 1
+        return max(couleurs) if couleurs else None
 
     @classmethod
-    def make_label(cls, vigilances, max_couleur=None):
-        if not isinstance(vigilances, list) or len(vigilances) < 1:
-            return ""
-        max_couleur = max_couleur or cls.make_max_couleur(vigilances)
-        if max_couleur == 1:
-            label = "Pas de vigilance météo"
-        else:
-            couleur = VigilanceMeteo.couleurs.get(max_couleur)
-            if couleur:
-                couleur = couleur.capitalize()
-            label = f"{couleur}"
-        return f"Vigilance météo: {label}"
+    def make_label(cls, max_couleur=None):
+        return VigilanceMeteo.labels.get(max_couleur) if max_couleur is not None else ''
```

Comment:
```suggestion
        return VigilanceMeteo.labels.get(max_couleur, '') ''
```

Reply:
ça fait la même chose, la deuxième valeur du `get` est la valeur par défaut

Reply:
Merci ! Python for dummies 😄